frontend/src/components/SeedConfiguration.vue

```diff
@@ -7,7 +7,7 @@ SPDX-License-Identifier: Apache-2.0
 <template>
   <action-button-dialog
     :shoot-item="shootItem"
-    :valid="valid"
+    :valid="seedClusterValid"
```

Comment:
I would prefer to stick to the name of the property in the shoot. 
Use `seedNameValid` instead of `seedClusterValid`

Reply:
I removed the validation logic as well as the component completely, as discussed